third_party/import-web-api/index.js

```diff
@@ -0,0 +1,72 @@
+import http from 'http';
+import { Form } from 'multiparty';
+import { exec } from 'child_process';
+import * as dotenv from 'dotenv'
+
+dotenv.config();
+
+const host = process.env.server_host;
+const port = process.env.server_port;
+const leaksDbFilePath = process.env.leaksdb_fp;
+
+const corsHeaders = {
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Methods': 'OPTIONS, POST',
+};
+
+const leakFormSchema = {
+    context: new String(),
+    leakFile: new String(),
+    shareDate: new Date(),
+    leakers: new String(),
+    platforms: new String(),
+};
+
+function triggerImportLeak(leakForm) {
+    const cmd = `./import --database-path="${leaksDbFilePath}" --leak-path="${leakForm.leakFile}" --context="${leakForm.context}" --platforms="${leakForm.platforms}" --share-date="${leakForm.shareDate}" --leakers="${leakForm.leakers}" --notify-url="https://subscribeService/notify"`;
```

Comment:
`--notify-url="https://subscribeService/notify"` should come from config

Reply:
done